users/views.py

```diff
@@ -15,10 +15,10 @@ def post(self, request, *args, **kwargs):
         return_value = super(UserRegistrationView, self).post(request=request)
         context = {
             'program': 'Backend',
```

Comment:
Please remove `program`, it is not related for this project. You can add this when it is required.